### PR TITLE
Feature: Support Syntax Highlighting

### PR DESCRIPTION
**Pull request by Qrbaker**
Here we are at last!

This change includes basic light and dark themes, and adds two config switches to enable everything:

* ~~`highlight-code: True` in the config YAML will turn on syntax highlighting~~
* ~~`highlight-dark: True` will, when `highlight-code` is set, use the dark mode css.~~

Update: highlighting is now always-on, and will use the theme preference as reported by the browser to select dark or light-mode CSS.

### CODE REVIEW

**Comment by Qrbaker** [comment]
RE: flake8. Its complaining about line length that black allows because black gives a little leeway on line length. They reccommend either increasing flake8's line-length by 10% (so 132 in our case) or tweaking flake8's config:

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

**Comment by TrueBrain** [comment]
Sorry for the slow reply .. again :(

I have been thinking about this PR, and I keep coming back to: should we make it optional at all? I am very tempted to say: no. TrueWiki is opinionated already, and that is exactly the point. What do you think?

You have a dark/light mode CSS stylesheet in this PR, but shouldn't that be bound to the browser-setting of the user visiting the website, rather than a global setting? I am somewhat tempted to change it into dark/light theme .. but I just wanted to check with you if that was actually your intention, or that you had some other ideas by supplying these two CSS.
I am a bit tempted to add dark/light theme suport in general, as that is a bit overdue for TrueWiki.

Let me know your opinions here :) Tnx!

**Comment by Qrbaker** [comment]
I'm okay making the code highlighting always-on. It would certainly make the code cleaner.

I hadn't actually thought of supporting dark/light mode. I'd probably recommend against it since the rest of the site CSS is not dynamic in that matter. I included both because I figured light mode would make sense with the default CSS, but I also made a dark CSS for the [Crafty wiki](https://wiki.craftycontrol.com), which is dark mode by default.

**Comment by Qrbaker** [comment]
@TrueBrain Okay, the PR now just has syntax highlighting enabled. Since the CSS is always used now, I combined it into the main `truewiki.css` file with Light/Dark mode media selectors.

File: static/truewiki/truewiki.css [hunk 1]
```diff
@@ -586,3 +586,177 @@ header {
     font-size: 18px;
     font-weight: bold;
 }
+
+/* Pygments - Light Mode */
```
Comment (TrueBrain) [inline static/truewiki/truewiki.css:590]: Please mark the source of these lines; can be something like: taken from pygments theme XXX

**Review by TrueBrain — APPROVED** [review]

**Pull request merged**